src/Microsoft.Extensions.Logging.EventLog/Internal/IEventLog.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Diagnostics;
+
+namespace Microsoft.Extensions.Logging.EventLog.Internal
+{
+    public interface IEventLog
+    {
+        int MaxMessageSize { get; }
```

Comment:
I actually found those tests a bit difficult to read :smile: It uses continuationString in some places but not others.

I think if it's going to go that route, I'd avoid "refactoring" the code too much and have the expected strings be more like:

```c#
"Test" + Environment.NewLine + "a...",
"...aaaaaa...",
"...aaaaaa...",
"...aaaaa",
```

Because that would really *show* the behavior, as opposed to *computing* the behavior.